src/main/java/io/pravega/connectors/flink/table/catalog/pravega/factories/PravegaCatalogFactory.java

```diff
@@ -100,24 +100,10 @@ public Catalog createCatalog(Context context) {
                 configOptions.get(PravegaCatalogFactoryOptions.SERIALIZATION_FORMAT));
 
         // put json related options into properties
-        properties.put(String.format("%s.%s",
-                        PravegaRegistryFormatFactory.IDENTIFIER, PravegaRegistryOptions.FAIL_ON_MISSING_FIELD.key()),
-                delegatingConfiguration.get(PravegaCatalogFactoryOptions.JSON_FAIL_ON_MISSING_FIELD).toString());
-        properties.put(String.format("%s.%s",
-                        PravegaRegistryFormatFactory.IDENTIFIER, PravegaRegistryOptions.IGNORE_PARSE_ERRORS.key()),
-                delegatingConfiguration.get(PravegaCatalogFactoryOptions.JSON_IGNORE_PARSE_ERRORS).toString());
-        properties.put(String.format("%s.%s",
-                        PravegaRegistryFormatFactory.IDENTIFIER, PravegaRegistryOptions.TIMESTAMP_FORMAT.key()),
-                delegatingConfiguration.get(PravegaCatalogFactoryOptions.JSON_TIMESTAMP_FORMAT));
-        properties.put(String.format("%s.%s",
-                        PravegaRegistryFormatFactory.IDENTIFIER, PravegaRegistryOptions.MAP_NULL_KEY_MODE.key()),
-                delegatingConfiguration.get(PravegaCatalogFactoryOptions.JSON_MAP_NULL_KEY_MODE));
-        properties.put(String.format("%s.%s",
-                        PravegaRegistryFormatFactory.IDENTIFIER, PravegaRegistryOptions.MAP_NULL_KEY_LITERAL.key()),
-                delegatingConfiguration.get(PravegaCatalogFactoryOptions.JSON_MAP_NULL_KEY_LITERAL));
-        properties.put(String.format("%s.%s",
-                        PravegaRegistryFormatFactory.IDENTIFIER, PravegaRegistryOptions.ENCODE_DECIMAL_AS_PLAIN_NUMBER.key()),
-                delegatingConfiguration.get(PravegaCatalogFactoryOptions.ENCODE_DECIMAL_AS_PLAIN_NUMBER).toString());
+        Map<String, String> jsonProperties = delegatingConfiguration.toMap();
+        jsonProperties.forEach((key, value) -> {
+            properties.put(String.format("%s.%s", PravegaRegistryFormatFactory.IDENTIFIER, key), value);
+        });
 
         PravegaConfig pravegaConfig = PravegaOptionsUtil.getPravegaConfig(configOptions);
         SchemaRegistryClientConfig schemaRegistryClientConfig = SchemaRegistryClientConfig.builder().
```

Comment:
This should use `SchemaRegistryUtils.getSchemaRegistryClientConfig` method to construct, otherwise credential information is not included.